src/main/java/org/mockito/Mock.java

```diff
@@ -117,10 +117,45 @@
     boolean lenient() default false;
 
     /**
-     * Mock will have custom strictness, see {@link MockSettings#strictness(Strictness)}.
+     * Mock will have custom strictness, see {@link MockSettings#strictness(org.mockito.quality.Strictness)}.
      * For examples how to use 'Mock' annotation and parameters see {@link Mock}.
      *
-     * @since 4.6.0
+     * @since 4.6.1
      */
-    Strictness strictness() default Strictness.STRICT_STUBS;
+    Strictness strictness() default Strictness.NOT_SET;
+
+    enum Strictness {
+
+        /**
+         * Default value used to indicate the mock does not override the test level strictness.
+         *
+         * @since 4.6.1
+         */
```

Comment:
I think naming it `TEST_LEVEL_DEFAULT` is a bit more descriptive and easier to understand for our users.

Reply:
Not sure I agree.  This value is explicitly saying that the `strictness` at the mock level is not set, hence `NOT_SET`.  IMHO, `NOT_SET` is more descriptive than `TEST_LEVEL_DEFAULT`.

However, I don't feel strongly about it.  So if you do I'm happy to change.

Reply:
Yeah I think `TEST_LEVEL_DEFAULT` is more descriptive for our users, so I would prefer to name it that.

Reply:
OK, done.